Marlin/language_ru.h

```diff
@@ -171,5 +261,110 @@
 #define MSG_DELTA_CALIBRATE_Y               _UxGT("Калибровать Y")
 #define MSG_DELTA_CALIBRATE_Z               _UxGT("Калибровать Z")
 #define MSG_DELTA_CALIBRATE_CENTER          _UxGT("Калибровать центр")
+#define MSG_DELTA_SETTINGS                  _UxGT("Показать настройки Delta")
+#define MSG_DELTA_AUTO_CALIBRATE            _UxGT("Авто калибровка")
+#define MSG_DELTA_HEIGHT_CALIBRATE          _UxGT("Задать высоту Delta")
+#define MSG_INFO_MENU                       _UxGT("О принтере")
+#define MSG_INFO_PRINTER_MENU               _UxGT("Информация о принтере")
+#define MSG_3POINT_LEVELING                 _UxGT("Выравнивание по 3 точкам")
+#define MSG_LINEAR_LEVELING                 _UxGT("Линейное выравнивание")
+#define MSG_BILINEAR_LEVELING               _UxGT("Квадратичное выравнивание")
+#define MSG_UBL_LEVELING                    _UxGT("Универсальное выравниваие")
```

Comment:
Before this can be accepted, every string that is over 20 characters must be shortened to 20 or fewer characters. You can use a ruler to check your strings like so:
```
12345678901234567890|
Правка сетки останов|лена
```